src/endpoints/kms.ts

```diff
@@ -17,13 +23,24 @@ enableEndpoint();
  */
 export const hearthbeat = (
   request: ccfapp.Request<void>,
-): ServiceResult<string | ccfapp.AuthnIdentityCommon> => {
+): ServiceResult<string | IHeartbeatResponse> => {
   const name = "hearthbeat";
   const serviceRequest = new ServiceRequest<void>(name, request);
 
   // check if caller has a valid identity
   const [policy, isValidIdentity] = serviceRequest.isAuthenticated();
   if (isValidIdentity.failure) return isValidIdentity;
 
-  return ServiceResult.Succeeded<ccfapp.AuthnIdentityCommon>(policy!);
+  const settings = Settings.loadSettings();
+  const description = [
+    settings.settings.service.name,
+    settings.settings.service.description,
```

Comment:
Maybe we want to put that `description` is a public information in README so that users don't accidentally put things that shouldn't be public?